src/main/java/org/folio/dao/transactions/BaseTemporaryTransactionsDAO.java

```diff
@@ -34,25 +34,25 @@ protected BaseTemporaryTransactionsDAO(String tableName) {
 
   @Override
   public Future<Transaction> createTempTransaction(Transaction transaction, String summaryId, String tenantId, Conn conn) {
-    if (transaction.getId() == null) {
+    logger.debug("Trying to create temp transaction");
+        if (transaction.getId() == null) {
       transaction.setId(UUID.randomUUID().toString());
     }
-
-    logger.debug("Creating new transaction with id={}", transaction.getId());
-
     try {
       return conn.execute(createTempTransactionQuery(tenantId),
         Tuple.of(UUID.fromString(transaction.getId()), pojo2JsonObject(transaction)))
         .map(transaction)
         .recover(ResponseUtils::handleFailure)
-        .onSuccess(x -> logger.debug("New transaction with id={} successfully created", transaction.getId()))
-        .onFailure(e -> logger.error("Transaction creation with id={} failed", transaction.getId(), e));
+        .onSuccess(x -> logger.debug("New transaction with id {} successfully created", transaction.getId()))
```

Comment:
Since we are using info log for success cases, we can change here to info as well.
Basically for post, put, delete requests I completely agree to use info for successful cases, for get requests - not sure what to use: debug or info, but lets wait other reviewers